src/services/event-service.ts

```diff
@@ -1,12 +1,84 @@
+import { v4 as uuidv4 } from "uuid";
+
 import { prisma } from "../utils/prisma";
+import logger from "../utils/logger/logger";
 
 class EventService {
-	static async createEvent() {
-		console.log("createEvent");
+	static async createEvent(
+		title: string,
+		description: string,
+		datetime: string,
+		duration: number,
+		point_value: number
+	) {
+		logger.info(
+			`EventService.createEvent invoked! title=${title} description=${description} date=${datetime} duration=${duration} point_value=${point_value} `
+		);
+
+		const UUID = uuidv4();
+
+		const formattedDatetime = new Date(datetime);
+
+		const event = prisma.event.create({
+			data: {
+				event_id: UUID,
+				title: title,
+				description: description,
+				datetime: formattedDatetime,
+				duration: duration,
+				point_value: point_value,
+			},
+		});
+
+		return event;
+	}
+
+	static async checkAttendance(uh_id: string, event_id: string) {
+		const attendance = await prisma.event_attendance.findFirst({
+			where: {
+				contact: {
+					uh_id,
+				},
+				event_id,
+			},
+		});
+
+		if (attendance) {
```

Comment:
This if/else block could be optimized with a simple:
```
return !!attendance;
```

Reply:
I didn't know that was possible. I will definitely use it in the future.